### DISCUSSION TITLE
skywalking docker-comose installation

### DISCUSSION THREAD

**Discussion by mr-vardhan24** [apache/skywalking#12074] [Q&A]
hi @wu-sheng
hope you are doing well.
i have deployed the skywalking using the docker-compose but I am not able to see the data over skywalking dashboard.

here is my docker-compose.yaml

`version: '3.3'
services:
elasticsearch:
image: docker.elastic.co/elasticsearch/elasticsearch:7.5.0
container_name: elasticsearch
restart: always
ports:
- 9200:9200
networks:
- demo
environment:
- discovery.type=single-node
- bootstrap.memory_lock=true
- "ES_JAVA_OPTS=-Xms256m -Xmx256m"
ulimits:
memlock:
soft: -1
hard: -1

oap:
image: apache/skywalking-oap-server:7.0.0-es7
container_name: oap
depends_on:
- elasticsearch
links:
- elasticsearch
restart: always
ports:
- 11800:11800
- 12800:12800
networks:
- demo
environment:
SW_STORAGE: elasticsearch7
SW_STORAGE_ES_CLUSTER_NODES: elasticsearch:9200

ui:
image: apache/skywalking-ui:7.0.0
container_name: ui
depends_on:
- oap
links:
- oap

networks:
  - demo
restart: always
ports:
  - 8080:8080
environment:
  SW_OAP_ADDRESS: oap:12800
otel-collector:
# Specify the image to start the container from
image: otel/opentelemetry-collector:0.19.0
# Set the otel-collector configfile
command: ["--config=/etc/otel-collector-config.yaml"]
# Mapping the configfile to host directory
volumes:
- ./otel-collector-config.yaml:/etc/otel-collector-config.yaml
ports:
- "13133:13133" # health_check extension
- "12800" # OpenCensus receiver
networks:
- demo

networks:
demo:
`

and here is the otel-collector-config.yaml

`extensions:
health_check:

A receiver is how data gets into the OpenTelemetry Collector
receivers:

Set Prometheus Receiver to collects metrics from targets
It’s supports the full set of Prometheus configuration
prometheus:
config:
scrape_configs:
- job_name: 'otel-collector'
scrape_interval: 10s
static_configs:
# Replace the IP to your VMs‘s IP which has installed Node Exporter
- targets: ['192.168.1.41:9100']
# - targets: ['vm2:9100']
#- targets: [‘vm3:9100']

processors:
batch:

An exporter is how data gets sent to different systems/back-ends
exporters:

Exports metrics via gRPC using OpenCensus format
opencensus:
endpoint: "host.docker.internal:12800" # The OAP Server address
retry_on_failure:
enabled: true
insecure: true
logging:
logLevel: debug

service:
pipelines:
metrics:
receivers: [prometheus]
processors: [batch]
exporters: [logging, opencensus]

extensions: [health_check]
`

and here the image of running every container
image

and here is the logs of the opentelemetry container

[otl-logs.txt](https://github.com/apache/skywalking/files/14865644/otl-logs.txt)

here is the oap server logs
[otl-logs.txt](https://github.com/apache/skywalking/files/14865968/otl-logs.txt)

**Comment by wu-sheng** [comment]
Don't repeatedly send things.
I don't have magic to know all things, especially it is on your side.

We are an OSS community, and volunteer based, we don't do things like commercial on call.
